scripts/install.js

```diff
@@ -3,16 +3,18 @@
 var download = require('download');
 var rimraf = require('rimraf');
 var semver = require('semver');
-var ProgressBar = require('progress')
+var ProgressBar = require('progress');
 var path = require('path');
-var fs = require('fs');
 var merge = require('merge');
 var urlModule = require('url');
 var Decompress = require('decompress');
 var fileExists = require('file-exists');
 var chalk = require('chalk');
 
-var buildType = process.env.npm_config_nwjs_build_type || process.env.NWJS_BUILD_TYPE || 'normal';
+var buildType =
+  process.env.npm_config_nwjs_build_type ||
+  process.env.NWJS_BUILD_TYPE ||
+  'normal';
```

Comment:
For readability. These should either all be on the same line or be nested in a parenthesis group where the start of the group appears on the first line and the end appears on the last line, like with curly braces.
```js
var buildType = (
  process.env.npm_config_nwjs_build_type ||
  process.env.NWJS_BUILD_TYPE ||
  'normal'
);
```
```suggestion
var buildType = process.env.npm_config_nwjs_build_type || process.env.NWJS_BUILD_TYPE || 'normal';
```

Reply:
Either way is fine with me